src/pages/waiting-room-to-car/components/bike-category-type/bike-category-type.ts

```diff
@@ -81,9 +87,24 @@ export class BikeCategoryTypeComponent implements OnChanges {
     this.testType  = (this.testType === BikeTestType.MOD1 || this.testType === BikeTestType.MOD2) ?
       this.testType : BikeTestType.MOD1;
     this.bikeCategoryDetails = this.bikeCategoryDetailProvider.getAllDetailsByTestType(this.testType);
+    this.initialCategory = this.testCategory;
```

Comment:
What happens when the selector is reopened? Does it not have the existing selected category?

Reply:
If the user reopens the selector the type remains selected but if the user exits the test and the reenters the selection is deselected.